gostream/codec/h264/ffmpeg/.gitignore

```diff
@@ -1,12 +0,0 @@
-# Include static libraries 
```

Comment:
I removed this and just did this in the Makefile since it's simpler and this was giving me issues.